includes/class-rest.php

```
@@ -189,6 +247,20 @@ public function clear_cache( \WP_REST_Request $request ) {
 			$params = $request->get_params();
 			$action = isset( $params['action'] ) ? sanitize_text_field( $params['action'] ) : '';
 			$path   = isset( $params['path'] ) ? sanitize_text_field( $params['path'] ) : '';
+			$group  = isset( $params['group'] ) ? sanitize_text_field( $params['group'] ) : '';
+
+			// Handle cache group flushing.
+			if ( ! empty( $group ) ) {
+				$flushed = Cache::flush_group( $group );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Cache' in method 'clear_cache'. (undefined)

(StaticAccess)